### PR TITLE
Sprint 21 TLT-1373 Adding huey worker to supervisor config

### PR DESCRIPTION
**Pull request by douglashall**
@bermudezjd 
@rascalking

### CODE REVIEW

**Comment by bermudezjd** [comment]
:+1:

File: supervisor.conf.j2 [hunk 1]
```diff
@@ -6,3 +6,8 @@ environment=DJANGO_SETTINGS_MODULE="{{ project_name }}.settings.{{ django_settin
 
 [group:{{ project_name }}]
 programs=gunicorn-{{ project_name }}
```
Comment (rascalking) [inline supervisor.conf.j2]: Doesn't the new huey-{{project_name}} program want to get added to this group?

Comment (douglashall) [inline supervisor.conf.j2]: Looking at this now. It seems to be failing with exceptions like the following on test:

File "/opt/tlt/lti_emailer/manage.py", line 13, in <module>
    execute_from_command_line(sys.argv)
  File "/var/opt/virtualenvs/lti_emailer/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/var/opt/virtualenvs/lti_emailer/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 330, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/var/opt/virtualenvs/lti_emailer/local/lib/python2.7/site-packages/django/core/management/base.py", line 390, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/var/opt/virtualenvs/lti_emailer/local/lib/python2.7/site-packages/django/core/management/base.py", line 441, in execute
    output = self.handle(*args, **options)
  File "/var/opt/virtualenvs/lti_emailer/local/lib/python2.7/site-packages/huey/djhuey/management/commands/run_huey.py", line 116, in handle
    self.autodiscover()
  File "/var/opt/virtualenvs/lti_emailer/local/lib/python2.7/site-packages/huey/djhuey/management/commands/run_huey.py", line 93, in autodiscover
    self.autodiscover_appconfigs()
  File "/var/opt/virtualenvs/lti_emailer/local/lib/python2.7/site-packages/huey/djhuey/management/commands/run_huey.py", line 63, in autodiscover_appconfigs
    app_path = config.module.__path__
AttributeError: 'module' object has no attribute '__path__'

Comment (bermudezjd) [inline supervisor.conf.j2]: Ah, @rascalking is correct.  You need to add the huey program to the project group.  Not clear from your comment if you're getting the above errors when you do that and deploy.

Comment (douglashall) [inline supervisor.conf.j2]: Yeah, I don't think the stack trace above is related to this. Submitted PR to add the Huey process to the group.

**Pull request merged**